app/helpers/hiring_staff/job_creation_helper.rb

```diff
@@ -1,14 +1,27 @@
 module HiringStaff::JobCreationHelper
   def current_step
-    params[:create_step]
+    step = params[:create_step]
```

Comment:
Can we reuse this helper in the application_controller to avoid repeating ourselves?